tests/screenobjects/components/carousel.js

```diff
@@ -51,7 +51,7 @@ class Carousel extends Gestures {
         const expectedText = partialText.toLowerCase();
 
         if (browser.isIOS) {
-            return expect(cardText).toContain(expectedText);
+            return global.expect(cardText).toContain(expectedText);
```

Comment:
Hmm, in the `browser.dynamic.spec.js` it's not a global. Personally I not fond of the global, can we make this generic over all files (remove global, add it to the eslint)?